app/src/main/res/layout/activity_login.xml

```diff
@@ -15,7 +15,7 @@
     <androidx.appcompat.widget.AppCompatImageView
         android:id="@+id/imageViewLogo"
         android:layout_width="wrap_content"
-        android:layout_height="160dp"
+        android:layout_height="200dp"
```

Comment:
Aren't you just increasing the height of the image view here? How does this solve the issue. Instead I think you should use the fill viewport attribute of Nested Scroll View.

Reply:
@epicadk do you think this is not working properly? @epicadk should this be done in the way you mentioned instead? cc @faznan3nazer